RdpCredProv/RdpCredential.cpp

```diff
@@ -7,7 +7,8 @@
 #include "RdpCredential.h"
 #include "guid.h"
 
-extern CLogFile log;
+extern CLogFile g_log;
```

Comment:
pointless rename that just makes the diff bigger and will break merging with upstream changes